src/profile/notification-settings/NotificationSettings.ts

```diff
@@ -0,0 +1,184 @@
+      h(UserAttributesCard, {
+        value: true,
+        label: 'Necessary communications related to platform operations',
+        setSaving,
+        notificationKeys: [],
+        notificationType: 'PlatformOperations',
+        disabled: true,
+      } as UserAttributesCardProps),
+      h(UserAttributesCard, {
+        value: marketingConsent,
+        label: 'Marketing communications (including upcoming workshops and new flagship dataset additions)',
+        setSaving,
+        notificationKeys: ['notifications/MarketingConsent'],
+        notificationType: 'Marketing',
+      } as UserAttributesCardProps),
+    ]),
+    div({ style: Style.cardList.toolbarContainer }, [
+      h2({ style: { ...Style.elements.sectionHeader, marginTop: '2rem', textTransform: 'uppercase' } }, [
+        'Account Notifications',
+        h(InfoBox, { style: { marginLeft: '0.5rem' } }, [
+          'You may receive email notifications regarding certain events in Terra. You may globally opt in or out of receiving these emails.',
+        ]),
+      ]),
+    ]),
+    h(NotificationCardHeaders),
+    div(
+      { role: 'list', 'aria-label': 'notification settings for your account', style: { flexGrow: 1, width: '100%' } },
+      [
+        h(NotificationCard, {
+          setSaving,
+          prefsData,
+          label: 'Group Access Requested',
+          options: [
+            {
+              notificationKeys: ['notifications/GroupAccessRequestNotification'],
+              notificationType: 'GroupAccessRequest',
+              optionLabel: 'Opt-in to group access requested notifications',
+            },
+          ],
+        } as NotificationCardProps),
+        h(NotificationCard, {
+          setSaving,
+          prefsData,
+          label: 'Workspace Access Added',
+          options: [
+            {
+              notificationKeys: ['notifications/WorkspaceAddedNotification'],
+              notificationType: 'WorkspaceAccessAdded',
+              optionLabel: 'Opt-in to workspace access added notifications',
+            },
+          ],
+        } as NotificationCardProps),
+        h(NotificationCard, {
+          setSaving,
+          prefsData,
+          label: 'Workspace Access Removed',
+          options: [
+            {
+              notificationKeys: ['notifications/WorkspaceRemovedNotification'],
+              notificationType: 'WorkspaceAccessRemoved',
+              optionLabel: 'Opt-in to workspace access removed notifications',
+            },
+          ],
+        } as NotificationCardProps),
+      ]
+    ),
+    div({ style: Style.cardList.toolbarContainer }, [
+      h2({ style: { ...Style.elements.sectionHeader, marginTop: '2rem', textTransform: 'uppercase' } }, [
+        'Workspace Notifications',
+        h(InfoBox, { style: { marginLeft: '0.5rem' } }, [
+          'You may receive email notifications related to workspace operations. You may opt in or out of receiving these emails for individual workspaces.',
```

Comment:
This message used to be specific to submissions, as it was the only thing you could toggle at a workspace level. Before: 'You may receive email notifications when a submission has succeeded, failed, or been aborted. You may opt in or out of receiving these emails for individual workspaces.'